### DISCUSSION TITLE
Crop Loss Damages

### DISCUSSION THREAD

**Discussion by HenryGeorgist** [USACE/go-consequences#38] [General]
How do crop losses work, what do they represent?

**Comment by HenryGeorgist** [comment]
@Tich1212 @Brennan1994 
From the original post we had this link:
https://github.com/USACE/go-consequences/blob/587076c011864c08b8cef383200a2e14e537c084/crops/crop.go#L121-L132

**Reply by HenryGeorgist** [reply]
The question here is why is loss equal to the damage factor times the _percentProductionValue_ times the totalMarketValueLessHarvestCost?

Why is the percent production value included in the calculation?

**Reply by HenryGeorgist** [reply]
The best answer I can give is that if the flood happens on the first day after planting, the farmer would not have experienced costs except for the seed and labor to put it in the ground. The farmer's losses would not be damage factor times total market value less harvest cost, it would be a percentage of that. What percentage? Well, the percentage of inputs included in the practice of producing that crop to the time of the flood. The best representation we have is the percentage of the total production value that has been input by the farmer at the date of the flood from their crop production function.

**Reply by Brennan1994** [reply]
I think where I'm getting hung up is actually on the definition of loss. It seems like we have two different types here combined in to one. 

1. Potential Profit: This is what I'm seeing total MarketValueLessHarvestCost as. This is the amount of money the farmer stands to make if he can sell the crop in the end. 

2. Cost of Business: This is the money the farmer puts into growing the crop. I think this is represented by the percentProductionValue

If the flood comes on day 1 and destroys 75% of the crop, I'd expect the losses to be 75% of the market value plus whatever he paid in gas/labor to plant and care for those crops the first day, but I don't think that's correct.

Assume Day 1 flood destroys 75% of the crop, and by day 1 Farmer Eric has put 10% of the work into that crop he expected to without the flood. Eric's losses are 75% of the market value multiplied by another 10% because he hadnt put that much work into it, so his loss is less. In my brain it should be slightly more, because .75 x marketvalue captures that loss of potential profit, and that work he put into it is an additional cost that should be added.

**Reply by HenryGeorgist** [reply]
Suppose you wanted to build a house, and after pouring the foundation and starting framing, you were flooded. what would your losses be? The market value of the home?

**Reply by HenryGeorgist** [reply]
> 1. Potential Profit: This is what I'm seeing total MarketValueLessHarvestCost as. This is the amount of money the farmer stands to make if he can sell the crop in the end.

This is not potential profit, it is potential earnings. Profit would be earnings minus costs.

**Reply by HenryGeorgist** [reply]
> 2\. Cost of Business: This is the money the farmer puts into growing the crop. I think this is represented by the percentProductionValue

Correct (with the exception of harvest costs)

**Reply by Brennan1994** [reply]
With the house example, that makes sense to me. But if I were calculating the losses in that case, I'd consider my losses to be just the labor and materials I put into the house to that point which I think is =~ exposedValue in the compute up there.  In our agricultural compute we have market value as part of the equation. If we're not considering potential earnings, then why is that market value part of the equation at all?

**Reply by Tich1212** [reply]
okay.

"The computation uses a proration of the total cost input less the harvest costs as a proxy for an exposed value."

To compute loss for the impacted case, we are not counting the work put in to the field from monthly and fixed cost as loss. We are only using them as a 'proxy' to characterize how far along the crop was before getting flooded?

**Reply by Brennan1994** [reply]
Ok. getting closer. With the house example. my losses would be whatever I spent on materials and labor to get it to the point it was before it flooded, plus whatever it's value was in that state.  and the value of it in that state would be whatever market value (MarketValueLessHarvestCost) of the completed thing was multiplied by some percentage of how far along in building it I was (percentProductionValue), muliplied again by however much of it I lost. Maybe 75% was destroyed (dmgFactor). 

I think the key here is me assigning value to a immature crop in my head. I was only seeing the value in a crop as being realized at harvest, since you can't really sell a bunch of tiny green tomatoes on the market. In this scenario though, we are assuming that the value of a crop is increasing everyday proportional to the work we've put into it. 

Am I close?

**Reply by HenryGeorgist** [reply]
I think @Tich1212 pointed out the important part,  that how value is characterized is critical to the discussion.

We are using the rate of inputs added to the field as a proxy for the creation of value. When does the farmer make the profit? At the time he plants the seed? At the time he delivers it to market? 

Within the context of this calculation I am suggesting my math pro-rates the value creation along the entirety of the production function at the rate of the value input to the field. 

We could simplify this by removing the market value, and instead multiplying by the inputs only (not converted to a percentage). This would suggest we do not carry the farmer's potential profit as part of the benefit base. This would be an understatement of the loss, but possibly the cleanest way to proceed, and probably most consistent with policy.

We could simplify this by removing the percentage of production (and always pay full price less harvest to the farmer) which would be an overstatement of the loss because there will be cases where the farmer did not bear the cost of some inputs to production, but yet received the benefit of loss.

Instead, the math (as it stands in the op) effectively operates between the two cases, providing a percentage of the profit based on the amount of input contributed. If the loss occurs on the day before harvest, the farmer would receive profit commensurate with the work, if it occurs the day after planting (to reflect lost opportunity to change course), they would receive only a small portion of the profit (to reflect the opportunity to change to a different crop).

**Reply by Brennan1994** [reply]
Ok. I feel like I understand now. I'm just going to try to repeat what you're saying dumbed down a little more in my words.

There's three ways to handle this problem. 
1.) Don't consider the value of the crops, just consider the cost of business. This would be multiplying the exposed value by the damage factor. It ignores the value of crops entirely. This is going to be an understatement of the loss. In this scenario we are assusming the value of the crops is 0 until they're harvested.   

2.) Pay full value for the crops. This would be taking market value times damage factor, and calling it good.  We're going to overcount cost though because the farmer didn't have to do the work of raising the crops that got damaged, but he's still gonna get credit for the value they would have had. In this scenario, we're assuming the value of the crops is full market value at planting. 

3.) Your approach is to prorate the value of the crops using the percentProduction. Meaning, the value of crops increases from 0 to full market value at the same rate the work on them is being completed. If you've spent 50% of your budget raising those crops, we assume those crops have a present value 50% of their market value.

**Reply by Brennan1994** [reply]
In our metaphor of the the house, I mused that the losses when it flooded would be the value of the house in it's current state, plus the labor and materials to get it there, multiplied by the damage factor (how much of it I lost ex. 75%)

In the ag compute, I see now that we're getting at the present value of those crops by prorating them with the production function,, but we're not actually accounting for the labor anywhere, correct? Labor is only included in the calculation as a stepping stone to figuring out the present worth of those crops at the time they flooded. Shouldn't we be adding a term for that somewhere?

**Reply by HenryGeorgist** [reply]
We are assuming that profit covers labor essentially.

**Reply by Brennan1994** [reply]
Alright. we have to use the marketPrice as part of the equation because we want account for the value of the immature crops somehow. We don't want to look at crops as having 0 value until their picked, rather that their value is increasing from 0 to their market value over time parellel to the work being put into the field. Loss is going to be the value of the crops at the time of flooding + the labor put into the field x the damage factor. There's not a good way to seperate the profit from the value of crops at the time of flooding, so we drop the second factor (labor put into the filed) and say it's a wash. 

Is that right?

**Comment by HenryGeorgist** [comment]
https://www.hec.usace.army.mil/confluence/agdocs/agriculture-damage-computation-proceedures-43820820.html

The link above is the current documentation on the computational procedures. There is always room for improvement.

**Reply by Brennan1994** [reply]
I will definitely contribute to that document once I've got a clear understanding
